src/date.js

```diff
@@ -63,6 +63,7 @@ Globalize.prototype.dateFormatter = function( options ) {
 	cldr.on( "get", validateRequiredCldr );
 	pattern = dateExpandPattern( options, cldr );
 	properties = dateFormatProperties( pattern, cldr );
+	properties.calendar = Globalize.calendars[cldr.attributes.calendar];
```

Comment:
All properties must be set in `dateFormatProperties`. Let's set `calendar` in there as well.